src/ui/public/chrome/directives/kbn_chrome.js

```diff
@@ -44,6 +46,9 @@ export default function (chrome, internals) {
         onRouteChange();
 
         // and some local values
+        $scope.$watchCollection(() => $http.pendingRequests, (pendingHttpRequests = []) => {
+          remove(pendingHttpRequests, isSystemApiRequest);
+        });
```

Comment:
Short isn't terse. In these cases I usually end up extracting both functions into their own named functions or variables.